### ISSUE TITLE
[suggestion] expose the copies attribute in the installer advanced disk configuration

### ISSUE THREAD

**Issue by RJVB** [trueos/trueos-core#1551]
Install targeting a laptop or small (all-in-one?) desktop system often have to make do with a single disk and cannot benefit from mirror devices. While it may be possible to achieve redundancy with multiple partitions on that same disk this won't give the same performance as using the copies dataset attribute (which can also be left to 1 on the datasets hosting non-crucial temp, cache, log etc. files).

I have already had the chance to be very grateful to using copies=2 on my ZoL system, when the laptop's started throwing bad sectors around. I didn't lose a single file.

In short, I think it's a missed chance not to be able to set the copies attribute in the installer's advanced disk configuration step. The attribute can be set later but that won't apply the new setting to files already installed.
